package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "kubewarden-js-policy-template",
+  "version": "0.1.0",
+  "description": "A JavaScript Kubewarden policy template",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "jest",
+    "e2e": "bats e2e.bats",
+    "format": "prettier --write .",
+    "format:check": "prettier --check ."
+  },
+  "dependencies": {
+    "kubewarden-policy-sdk": "file:../policy-sdk-js/js/kubewarden-policy-sdk-0.1.0-alpha.tgz"
```

Comment:
Just a friendly reminder, we need to update this to the npm released package in the future.